CHANGELOG.md

```diff
@@ -1,22 +1,32 @@
-_With the release of `v3.0.0`, we're introducing a new changelog format in an attempt to consolidate the information presented in the changelog. The new changelog is reduced in scope to only documenting functionality changes from version to version. This ensures that the changelog is as useful as it can be. Changes which should be documented include:_
+# Singularity Changelog
 
-  - _Renamed commands_
-  - _Deprecated / removed commands_
-  - _Changed defaults / behaviors_
-  - _Migration guidance_
-  - _New features / functionalities_
+## v3.8.1 - [2021-08-12]
 
+### Changed defauls / behaviours
 
-_The old changelog can be found in the `release-2.6` branch_
+  - `build-singularity` example pulls the base image from Docker/RockyLinux 
+    instead of Docker/CentOS
+  - Set LABELs properly in the container when pulling from Docker/OCI source.
+    Use --force in a build to override a label that already exists in source.
+  - Partially replace the github.com/satori/go.uuid module.
 
+### Bug Fixes
 
-# v3.8.0 - [2021-06-15]
+  - Allow escaped `\$` in a SINGULARITYENV_ var to set a literal `$` in
```

Comment:
Also please add a note for changed functionality in #6008.  It's in some ways similar to this first note so I would put it first or second.